imagersite/imagersite/templates/registration/logout.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<h3>Successfully logged out.</h3>
+
+<a href="../../">Back to home</a>
```

Comment:
Why write your URLs manually when you have names for your URLs and access to Django's `{% url %}` template tag?